src/cbmc/cbmc_parse_options.cpp

```diff
@@ -107,6 +110,20 @@ void cbmc_parse_optionst::set_default_options(optionst &options)
   options.set_option("depth", UINT32_MAX);
 }
 
+void cbmc_parse_optionst::set_soundness_on_by_default(optionst &options)
```

Comment:
Please avoid the word "soundness" in this context. It has a specific meaning, and not very much to do with those flags.

Reply:
I agree.

I suggest to change it to `set_thorough_by_default`.